promql/engine.go

```diff
@@ -736,7 +736,7 @@ func (ev *evaluator) vectorSelector(node *VectorSelector) Vector {
 		}
 		vec = append(vec, Sample{
 			Metric: node.series[i].Labels(),
-			Point:  Point{V: v, T: ev.Timestamp},
+			Point:  Point{V: v, T: t},
```

Comment:
This breaks graphing. The reason why we align timestamps in vectors to the evaluation timestamp is that graphing frontends will have a difficult time aligning things by themselves, and then instead of:

![good](https://cloud.githubusercontent.com/assets/538008/25047030/dd1c92ee-2134-11e7-8286-e97938d6872f.png)

...you will get:

![broken](https://cloud.githubusercontent.com/assets/538008/25047046/fe8d7e98-2134-11e7-90cd-291a3ac0f8e0.png)